### DISCUSSION TITLE
官网的文档和代码不一致啊！！！！

### DISCUSSION THREAD

**Discussion by larrygogo** [ant-design/ant-design#38936] [Q&A]
版本 5.0.1

![819a3c4b-59af-46df-984d-269fbf785c37](https://user-images.githubusercontent.com/15634028/203685692-f5441940-96c7-4b1f-854d-6051bace57ad.jpeg)
![vn3hQEqE5E](https://user-images.githubusercontent.com/15634028/203685700-2c27f084-af11-4d83-95e6-fb615ea0eb50.jpg)

**Comment by MadCcc** [comment]
收到反馈，我们会更新一下
